ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -73,21 +168,27 @@ const Footer = () => {
       >
         {t('cancel')}
       </Button>
-      <Button
-        block
-        data-testid="confirm-footer-confirm-button"
-        onClick={onSubmit}
-        size={ButtonSize.Lg}
+      {alertModalVisible ? (
```

Comment:
Since the modal and all the above hooks are only relevant to the confirm button, should we encapsulate all that within our `ConfirmFooterButton` component to keep this main render as simple as possible?

Reply:
Done.